tests/test_chef_integration.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+from le_utils.constants import licenses
```

Comment:
I was rather expecting us to just edit the existing tutorial chef: https://github.com/learningequality/ricecooker/blob/develop/examples/tutorial/sushichef.py and run that as an integration test, to avoid duplicaton.

Reply:
The actual sushichef file has the template strings and if you run a chef with the template strings in the config it won't run.

Alternatively, we could remove that check on running the chef. 

In any case, I figured that something that is a test is liable to be extended with more robust testing scenarios that wouldn't be relevant to the tutorial chef.